ivy/functional/ivy/manipulation.py

```diff
@@ -807,6 +823,30 @@ def repeat(
         b: ivy.array([3., 3., 4., 4., 5., 5.])
     }
 
+    With a mix of :code:`ivy.Array` and :code:`ivy.Container` inputs:
```

Comment:
Seems like you've added the docstring for the `clip` function in the `repeat` function, please revert this change.